AlphaWallet/Tokens/ViewModels/EthCurrencyHelper.swift

```diff
@@ -24,6 +24,14 @@ class EthCurrencyHelper {
         }
     }
 
+    var marketPrice: Double? {
+        if let price = ticker?.price_usd {
```

Comment:
Just this will do, right?

`return ticker?.price_usd`

Reply:
Done.